src/main/java/ubb/proiectColectiv/businessmanagementbackend/controller/SupervisorController.java

```diff
@@ -24,8 +31,14 @@ public SupervisorController(SupervisorService supervisorService) {
     }
 
     @GetMapping(value = "/supervisor/requests")
-    public ResponseEntity<String> get_Requests(@RequestBody String content) {
-        return null;
+    public ResponseEntity<String> getRequests() {
+        try {
+            logger.info("Sending all unapproved users!\n");
```

Comment:
logger.info already adds new line at the end. For consistency inside the console log II would like to ask you to remove the "\n"

Reply:
Solved.